### ISSUE TITLE
Feature request : Send an OSC message OSCInterface.on_exit()

### ISSUE THREAD

**Issue by stefets** [mididings/mididings#5]
Hi!

I have inherit the LiveOSC server class in my Flask/HTML5 UI for mididings [flaskdings](https://github.com/stefets/flaskdings) and I would like to know when the mididings OSC interface is shutting down; for instance CTRL-C.

I suggest to send an OSC message when the OSC interface will be stopped; with that message, any OSC server connected to mididings would be notify of the shutdown.

As a use case, flaskdings service will notify all the web clients and the HTML page will change the status of the engine from the state of running to not running.

Something like :

```
def on_exit(self):
    if self.port is not None:
        _liblo.send(p, '/mididings/exit') # <----
        self.server.stop()
        del self.server
```

**Comment by grawlinson** [comment]
Is the below code you're talking about?

https://github.com/mididings/mididings/blob/75ff8e44f62aa5ab246f6391fdf2b69c321002c0/mididings/extra/osc.py#L61-L64

Should be simple enough (famous last words!), I'll have a look but if you want to submit a PR, go for it :)

**Comment by stefets** [comment]
> Is the below code you're talking about?

Yes it is! Ok I will submit a PR for this. 

I will submit another PR later to add 3 methods that shoud be in LiveOSC; 

You can add me to the assignees if you want. Tx!

**Comment by grawlinson** [comment]
Awesome, thanks! :)

**Comment by grawlinson** [comment]
Closed in PR #6.
